boards/common/microbit/microbit.c

```diff
@@ -137,7 +137,7 @@ static const uint8_t pixmap[ROWS][COLS] = {
 /**
  * @brief   Timer dev
  */
-#define TIMER_DEV_NUM       (1)
+#define TIMER_DEV_NUM       (2)
```

Comment:
The `microbit` (first version) has a `nrf51`but it also seems to have 3 timer, did you by any change test on that version as well?

Reply:
Yes, tested with both (that's how I found that the additional `void TIMER_3_ISR` is needed).

Reply:
Ok, its always better is test output is shown In the PR, but I guess since in this case the output is graphical it doesn't make that much sense.